python3/poweron/power-on.py

```diff
@@ -26,8 +26,8 @@ def waitForXapi(session, host):
         metrics = session.xenapi.host.get_metrics(host)
         try:
             finished = session.xenapi.host_metrics.get_live(metrics)
-        except:
-            pass
+        except Exception as e:
+            print(f"Exception occurred: {e}")
```

Comment:
Some remarks:
- Errors should usually be logged to stderr, not stdout.
- We can log the type name of the exception
- There is no need to use an f-string, but of course you may if you really think that it is better.

```suggestion
            import sys
            print(type(e).__name__, " occured:", e, file=sys.stderr)
```

Of course:
- move the `import sys` to the top of the file, and
- run `isort` on the file to sort the imports.

Example for testing:
```py
import sys

try:
    raise OSError("Error message from the Exception")
except Exception as e:
    print(type(e).__name__, "occurred:", e, file=sys.stderr)
```